worlds/mm-recomp/Rules.py

```diff
@@ -2,11 +2,6 @@
 
 from BaseClasses import CollectionState, MultiWorld
 
-
-def universal_item_rule(item):
-    pass
-
-
```

Comment:
lol i need this, i'm gonna use it for keysanity

Reply:
That's not even in the copy I originally downloaded, so however that got deleted wasn't by me. But can add it back in for ya